### PR TITLE
Add test case when the integer value is negative

### LINKED ISSUE #286: [compiler] Add test case when the integer value is negative
**Subject**
compiler

**Detail**
Add test case when the integer value is negative in `TestCompileIntegerLiteral()`

### PR DESCRIPTION
**Pull request by AgwaB**
resolved: #286

Add test case when the integer value is negative

### CODE REVIEW

File: translate/compiler_internal_test.go [hunk 1]
```diff
@@ -787,6 +787,31 @@ func TestCompileIntegerLiteral(t *testing.T) {
...
+		},
+		{
+			expression: &ast.IntegerLiteral{
+				Value: -20,
+			},
+			expected: Bytecode{
+				RawByte: []byte{0x21, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xEC},
+				AsmCode: []string{"Push", "ffffffffffffffec"},
+			},
+		},
```
Comment (boohyunsik) [inline translate/compiler_internal_test.go:809]: If -0? enable?

Comment (AgwaB) [reply translate/compiler_internal_test.go:809]: Yes. Recognize as 0.

**Review by hihiboss — APPROVED** [review]
TYVM!
